Core/include/Acts/EventData/detail/CorrectedTransformationFreeToBound.hpp

```diff
@@ -0,0 +1,41 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2021-2022 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Definitions/Common.hpp"
+#include "Acts/EventData/detail/TransformationFreeToBound.hpp"
+
+namespace Acts {
+namespace detail {
+
+/// @brief Parameters sampling based on covariance matrix sqrt root in UKF: https://doi.org/10.1117/12.280797
+///
+///@todo make the parameters configurable
+struct CorrectedFreeToBoundTransformer {
```

Comment:
The Idea here is to construct this 
`CorrectedFreeToBoundTransform{0.2,4.}` I guess?

Reply:
I have added the constructor using the parameters explicitly.

Reply:
Yes, please move it out of the `detail` namespace though.

Reply:
I have removed the `FreeToBoundCorrection` out of the `detail`.